tmt/utils.py

```diff
@@ -1585,16 +1586,16 @@ def dict_to_yaml(
         data: Union[Dict[str, Any], List[Any], 'tmt.base._RawFmfId'],
         width: Optional[int] = None,
         sort: bool = False,
-        start: bool = False) -> str:
+        start: Optional[bool] = False) -> str:
     """ Convert dictionary into yaml """
     output = io.StringIO()
     yaml = YAML()
     yaml.indent(mapping=4, sequence=4, offset=2)
     yaml.default_flow_style = False
     yaml.allow_unicode = True
     yaml.encoding = 'utf-8'
-    yaml.width = width
-    yaml.explicit_start = start
+    yaml.width = width  # type: ignore[assignment]
+    yaml.explicit_start = start  # type: ignore[assignment]
```

Comment:
These look more like an issue in ruamel?

Reply:
Yeah, mypy infers types based on what's assigned to these two fields, and that's probably `None` in some `__init__()` somewhere in ruamel, hence mypy has no notion of these fields being `Optional[int]`. Please, add a comment like https://github.com/teemtee/tmt/blob/main/tmt/steps/discover/shell.py#L124 & blame ruamel.yaml.